### DISCUSSION TITLE
SCRAM-SHA-256-PLUS

### DISCUSSION THREAD

**Discussion by billglennon** [rapidloop/pgmetrics#51] [Q&A] [answered]
Hi,

We started using the password encryption "scram-sha-256" in our PostgreSQL database (version 13.2 ) for our database users.

Now we are not able to use pgmetrics (version 1.11.0).  We get the following error message => pgmetrics: pq: unknown sasl mechanism: "SCRAM-SHA-256-PLUS".

Any ideas or suggestions to look at? 

Thanks!

**Comment by mdevan** [comment]
Are you using stock PostgreSQL (from official PostgreSQL APT/YUM repos or downloaded from postgresql.org) or a vendor / cloud provider distribution?

**Reply by billglennon** [reply]
Using PostgreSQL 13.2 AWS RDS.

**Comment by mdevan** [comment, marked as answer]
The just-committed revision 444804f should fix this. You'll need to build pgmetrics from master to try it out:

(Install Go first if you don't have it -- https://golang.org/dl/)

```
git clone https://github.com/rapidloop/pgmetrics
cd pgmetrics
go build ./cmd/pgmetrics
```

Let us know how it goes.

**Reply by billglennon** [reply]
Thanks! I will pick this up on Monday and try and test it. Thanks for your help!

**Reply by billglennon** [reply]
Hi, 
I tested it and it worked fine. 
Thanks!

**Reply by billglennon** [reply]
Hi,
Just wondering, will you be doing a new release with this fix? 
Thanks again for your help and quick response on this! Very much appreciated!

**Reply by mdevan** [reply]
Yes, there should be a new release sometime during this month (October).

**Reply by billglennon** [reply]
Great! Thank you!

**Reply by mdevan** [reply]
This is now available in v1.12.

**Reply by billglennon** [reply]
Great! Thanks!
